src/music_album.rb.rb

```diff
@@ -1,9 +1,9 @@
 require_relative 'item'
 
-class Music_album < Item
+class MusicAlbum < Item
   attr_accessor :on_spotify
 
```

Comment:
Can you fix the file name here there are two rb in the end

Reply:
Thanks for the suggestion. This has been fixed